shared/chat/conversation/attachment-popup/index.native.js

```diff
@@ -31,7 +32,14 @@ class AutoMaxSizeImage extends Component<any, {width: number, height: number}> {
   }
 
   render() {
-    return (
+    return isAndroid ? (
```

Comment:
can't we just use this for ios too?

Reply:
we could, but IMO it's not as smooth as the scrollView on iOS and there are some behavior differences that make it worse. we can talk about it if you want